### DISCUSSION TITLE
Thank you so MUCH!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by cjlaborde** [PabloLec/RecoverPy#89] [Q&A] [answered]
You helped me recover a file of over 4000 lines....
Not sure if file is complete but most of what I remember is here.

This apps works and perform what it does.

My issue was that  I somehow save file when file was empty and when I reopen file was empty...
I was able to recover some part of the content.

Your apps is great, would be better if you made youtube video explaining it since the UI is kind of rough to use.
For those who have problems you need to use arrow keys to select the area you interact with or you going to get error error message that partition is not open.

**Comment by PabloLec** [comment, marked as answer]
Glad it helped you @cjlaborde :wink: :+1:

**Comment by PabloLec** [comment]
@cjlaborde Let me know if you had any trouble in particular using the tool, especially with the UI.
I understand that mouse support didn't work in your case ?
